app/src/main/java/nsit/app/com/nsitapp/AboutUs.java

```diff
@@ -43,49 +71,29 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.clear();
         super.onCreateOptionsMenu(menu, inflater);
     }
-
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         //setHasOptionsMenu(false);
         View rootView = inflater.inflate(R.layout.fragment_aboutus, container, false);
-        ImageView s1, s2, s3, s4, s5, s6, s8, s9, s10, s11,s12,s13, namanFB, namanLinkedIn, prabhakrFB, prabhakrLinkedIn;
-        ImageView gitsidharth, gitswati, gitprabhakr, gitnaman, gitsagar, gitsumit,prabhakrim,gitayush;
-
-
-        TextView con;
-        TextView rep;
-        s1 = (ImageView) rootView.findViewById(R.id.linked_sidharth);
-        s2 = (ImageView) rootView.findViewById(R.id.linked_swati);
-        s3 = (ImageView) rootView.findViewById(R.id.facebook_sidharth);
-        s4 = (ImageView) rootView.findViewById(R.id.facebook_swati);
-        s8 = (ImageView) rootView.findViewById(R.id.facebook_sumit);
-        s9 = (ImageView) rootView.findViewById(R.id.linked_sumit);
-        s5 = (ImageView) rootView.findViewById(R.id.facebook_chetan);
-        s6 = (ImageView) rootView.findViewById(R.id.linked_chetan);
-        s12 = (ImageView) rootView.findViewById(R.id.facebook_ayush);
-        s13 = (ImageView) rootView.findViewById(R.id.linked_ayush);
-
+        ButterKnife.bind(this,rootView);
 
         prabhakrFB = (ImageView) rootView.findViewById(R.id.facebook_prabhakr);
         prabhakrLinkedIn = (ImageView) rootView.findViewById(R.id.linked_prabhakr);
-
         gitsidharth = (ImageView) rootView.findViewById(R.id.github_sidharth);
         gitswati = (ImageView) rootView.findViewById(R.id.github_swati);
         gitprabhakr = (ImageView) rootView.findViewById(R.id.github_prabhakr);
         gitsagar = (ImageView) rootView.findViewById(R.id.github_sagar);
         gitnaman = (ImageView) rootView.findViewById(R.id.github_naman);
         gitsumit = (ImageView) rootView.findViewById(R.id.github_sumit);
         gitayush = (ImageView) rootView.findViewById(R.id.github_ayush);
-
         prabhakrim = (ImageView) rootView.findViewById(R.id.image_prabhakr);
-
         s10 = (ImageView) rootView.findViewById(R.id.linked_sagar);
         s11 = (ImageView) rootView.findViewById(R.id.facebook_sagar);
-
         con = (TextView) rootView.findViewById(R.id.cont);
         namanFB = (ImageView) rootView.findViewById(R.id.facebook_naman);
         namanLinkedIn = (ImageView) rootView.findViewById(R.id.linked_naman);
         rep = (TextView) rootView.findViewById(R.id.GoToRepo);
+
         s1.setOnClickListener(new View.OnClickListener() {

```

Comment:
Don't worry about the Travis test right now.
It was failing earlier also. Somehow it is because I am using android-25 build tools.